### ISSUE TITLE
block breaks moodle quiz in Mdl 2.4.1

### ISSUE THREAD

**Issue by bmbrands** [marxjohnson/moodle-block_accessibility#17]
Hi there,

I noticed the accessibility block causes an error when submitting answers in a Moodle quiz. In a Moodle page I get the error "Cannot require a CSS file after <head> has been printed.

I found this in my apache logs:

[Sun Mar 03 12:34:41 2013] [error] [client 127.0.0.1] Default exception handler: Coding error detected, it must be fixed by a programmer: Cannot require a CSS file after &lt;head&gt; has been printed. Debug: /blocks/accessibility/userstyles.php\nError code: codingerror\n\* line 568 of /lib/outputrequirementslib.php: coding_exception thrown\n\* line 52 of /blocks/accessibility/block_accessibility.php: call to page_requirements_manager->css()\n\* line 118 of /blocks/moodleblock.class.php: call to block_accessibility->init()\n*
line 1599 of /lib/blocklib.php: call to block_base->__construct()\n\* line 2592 of /lib/pluginlib.php: call to block_instance()\n\* line 36 of /admin/settings/plugins.php: call to plugininfo_block->load_settings()\n\* line 6264 of /lib/adminlib.php: call to require()\n\* line 3270 of /lib/navigationlib.php: call to admin_get_root()\n\* line 3175 of /lib/navigationlib.php: call to settings_navigation->load_administration_settings()\n\* line 701 of /lib/pagelib.php: call to settings_navigation->initialise()\n\* line 717 of /lib/pagelib.php: call to moodle_page->magic_get_settingsnav()\n\* line 2975 of /lib/navigationlib.php: call to moodle_page->__get()\n\* line
2645 of /lib/outputrenderers.php: call to navbar->get_items()\n\* line 119 of /theme/formal_white/layout/general.php: call to core_renderer->navbar()\n\* line 804 of /lib/outputrenderers.php: call to include()\n\* line 734 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()\n\* line ? of unknownfile: call to core_renderer->header()\n\* line 234 of /lib/outputrenderers.php: call to call_user_func_array()\n\* line 554 of /mod/quiz/renderer.php: call to plugin_renderer_base->__call()\n\* line 554 of /mod/quiz/renderer.php: call to mod_quiz_renderer->header()\n\* line 99 of /mod/quiz/summary.php: call to mod_quiz_renderer->summary_page()\n
[Sun Mar 03 12:35:24 2013] [error] [client 127.0.0.1] Default exception handler: Coding error detected, it must be fixed by a programmer: Cannot require a CSS file after &lt;head&gt; has been printed. Debug: /blocks/accessibility/userstyles.php\nError code: codingerror\n\* line 568 of /lib/outputrequirementslib.php: coding_exception thrown\n\* line 52 of /blocks/accessibility/block_accessibility.php: call to page_requirements_manager->css()\n\* line 118 of /blocks/moodleblock.class.php: call to block_accessibility->init()\n*
line 1599 of /lib/blocklib.php: call to block_base->__construct()\n\* line 2592 of /lib/pluginlib.php: call to block_instance()\n\* line 36 of /admin/settings/plugins.php: call to plugininfo_block->load_settings()\n\* line 6264 of /lib/adminlib.php: call to require()\n\* line 3270 of /lib/navigationlib.php: call to admin_get_root()\n\* line 3175 of /lib/navigationlib.php: call to settings_navigation->load_administration_settings()\n\* line 701 of /lib/pagelib.php: call to settings_navigation->initialise()\n\* line 717 of /lib/pagelib.php: call to moodle_page->magic_get_settingsnav()\n\* line 2975 of /lib/navigationlib.php: call to moodle_page->__get()\n\* line
2645 of /lib/outputrenderers.php: call to navbar->get_items()\n\* line 119 of /theme/formal_white/layout/general.php: call to core_renderer->navbar()\n\* line 804 of /lib/outputrenderers.php: call to include()\n\* line 734 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()\n\* line ? of unknownfile: call to core_renderer->header()\n\* line 234 of /lib/outputrenderers.php: call to call_user_func_array()\n\* line 554 of /mod/quiz/renderer.php: call to plugin_renderer_base->__call()\n\* line 554 of /mod/quiz/renderer.php: call to mod_quiz_renderer->header()\n\* line 99 of /mod/quiz/summary.php: call to mod_quiz_renderer->summary_page()\n, referer: http://clue/
